splunk_setup/dashboard.html

```diff
@@ -0,0 +1,70 @@
+<dashboard version="1.1" theme="light">
```

Comment:
As we will likely have other dashboards in the future, it would be great for this to have a more descriptive name and some documentation or description on what this dashboard contains.